### DISCUSSION TITLE
Cant connect with remote host

### DISCUSSION THREAD

**Discussion by sohag033** [dani-garcia/vaultwarden#2308] [Q&A]
### I want to connect the remote database with PGSQL. I have already tried with just docker with below cmd and its working fine
docker run -d --name vaultwarden -e DATABASE_URL=postgresql://vaultwarden:password@db host:5432/vaultwarden -p 80:80 vaultwarden/server:latest

### But in K8s I am trying to connect to the remote host with the below configmap. why it’s not connected?
apiVersion: v1
kind: ConfigMap
metadata:
name: vaultwarden-configmap
namespace: sohag
data:
DATABASE_CONNECTION: “postgresql”
DATABASE_URL: “[host.rds.com:5432](http://host.rds.com:5432/)”
POSTGRES_USER: “vaultwarden”
POSTGRES_PASSWORD: “Password”
POSTGRES_DB: “vaultwarden”

### could you please help me to solve the problem?
BR,
Samajpati

**Comment by Masgalor** [comment]
I have no idea what "K8s" is so this might not be helpfull, but I will give it a try anyway.

The variables in your configuration make no sense, vaultwarden does not support:
```
DATABASE_CONNECTION
POSTGRES_USER
POSTGRES_PASSWORD
POSTGRES_DB
```

Why dont you use the database-url the same way you did in your testrun with docker?
[Reference Database URL](https://github.com/dani-garcia/vaultwarden/blob/08f0de7b46d36a4e974d8e7b25a9786168cd38a5/.env.template#L16-L21)
[Wiki Postgresql](https://github.com/dani-garcia/vaultwarden/wiki/Using-the-PostgreSQL-Backend)
